### PR TITLE
multithreded context

### PR DESCRIPTION
**Pull request by Nanopro**
Can't share Cell<bool> between threads, replace Cell<bool> with AtomicBool.

### CODE REVIEW

**Comment by Ralith** [comment]
Is there strong reason to believe that all API calls are, in fact, thread-safe?

**Comment by Nanopro** [comment]
I want to use Context as a Resource in my ECS app (specs, shred), but i can't because Cell is not Send.

**Comment by Ralith** [comment]
Introducing safety hazards isn't a good solution to that. Try passing it into a system that you run manually (not with the dispatcher) instead of making it a resource.

**Comment by Nanopro** [comment]
I don't see any reasons not to use AtomicBool in context, becouse System and Compositor are Send, and you are able to make API calls from different threads. Replacing Cell<bool> with AtomicBool does not introduce safety hazards.

File: Cargo.toml [hunk 1]
```diff
@@ -22,5 +22,5 @@ travis-ci = { repository = "rust-openvr/rust-openvr" }
 maintenance = { status = "passively-maintained" }
 
 [dependencies]
-openvr_sys = "2"
+openvr_sys = "=2.0.2"
```
Comment (Ralith) [inline Cargo.toml]: An `=` constraint here is not appropriate.

**Pull request merged**